src/services/member-service.ts

```diff
@@ -1,15 +1,40 @@
+// import { type } from "os";
 import logger from "../utils/logger/logger";
 
-// import SQLService from "./sql-service";
+import SQLService from "./sql-service";
 
 class MemberService {
 	static async isMember(student_id = "", email = "") {
```

Comment:
For simplicity's sake, it may be easier to have just one function that 1. check's someone's membership and 2. returns member status, start, and end date. This is because we're pulling all of this information in the same SQL selection from the `member` table.

Reply:
> For simplicity's sake, it may be easier to have just one function that 1. check's someone's membership and 2. returns member status, start, and end date. This is because we're pulling all of this information in the same SQL selection from the `member` table.

Yeah that sounds good. In that case should I keep the function name 'isMember()' or change it to something like 'getMembership()' or 'membershipStatus()'?